api/proto/common/common.proto

```diff
@@ -11,19 +13,24 @@ message G1Commitment {
 }
 
 // BlobCommitment represents commitment of a specific blob, containing its
-// KZG commitment, degree proof, the actual degree, and data length in number of symbols.
+// KZG commitment, degree proof, the actual degree, and data length in number of symbols (field elements).
+// It deserializes into https://github.com/Layr-Labs/eigenda/blob/ce89dab18d2f8f55004002e17dd3a18529277845/encoding/data.go#L27
+//
+// See https://github.com/Layr-Labs/eigenda/blob/master/docs/spec/attestation/encoding.md#validation-via-kzg
+// to understand how this commitment is used to validate the blob.
 message BlobCommitment {
-  // A commitment to the blob data.
+  // Concatenation of the x and y coordinates of `common.G1Commitment`.
   bytes commitment = 1;
   // A commitment to the blob data with G2 SRS, used to work with length_proof
-  // such that the claimed length below is verifiable.
+	// such that the claimed length below is verifiable.
   bytes length_commitment = 2;
   // A proof that the degree of the polynomial used to generate the blob commitment is valid.
-  // It is computed such that the coefficient of the polynomial is committing with the G2 SRS
-  // at the end of the highest order.
+	// It is computed such that the coefficient of the polynomial is committing with the G2 SRS
```

Comment:
Can we just say like it's `a commit to x^(SRSOrder-n) * P(x), where P(x) is polynomial of degree n representing the blob`?  This is not only clear, but concise.

Reply:
[dd108ef](https://github.com/Layr-Labs/eigenda/pull/1104/commits/dd108efcb8080a5df28cb159d906535a9740e098)